File: lib/android/app/src/test/java/com/reactnativenavigation/controllers/bottomtabs/attachmode/TogetherTest.java
package com.reactnativenavigation.controllers.bottomtabs.attachmode;

import com.reactnativenavigation.controllers.bottomtabs.*;
import com.reactnativenavigation.controllers.viewcontroller.ViewController;

import org.junit.*;

public class TogetherTest extends AttachModeTest {

    @Override
    public void beforeEach() {
        super.beforeEach();
        uut = new Together(parent, tabs, presenter, options);
    }

    @Test
    public void attach_allTabsAreAttached() {
        uut.attach();
        assertIsChild(parent, tabs.toArray(new ViewController[0]));
    }
}

File: android/src/androidTest/java/com/reactnativenavigation/viewcontrollers/bottomtabs/attacher/modes/TogetherTest.kt
package com.reactnativenavigation.viewcontrollers.bottomtabs.attacher.modes

import org.junit.Before
import org.junit.Test

class TogetherTest : AttachModeTest() {

    @Before
    override fun setup() {
        super.setup()
        uut = Together(parent, tabs, presenter, options)
    }

    @Test
    fun attach_allTabsAreAttached() {
        uut.attach()
        assertIsChild(parent, *tabs.toTypedArray())
    }
} 